src/components/MapController/MapController.js

```diff
@@ -150,9 +150,23 @@ var MapController = React.createClass({
           onSetArea={this.handleSetArea}
           area={this.state.area}
         />
+      );    
+      var dataAttributes = this.state.dataset.split('_');
+      var run = dataAttributes[4];
+      var startYear = dataAttributes[5].slice(0, 4);
+      var endYear = dataAttributes[5].slice(9, 13);
+      var time = this.state.wmstime.slice(0,10);
```

Comment:
Definitely good to display the time that we're looking at, but why ``slice(0,10)``? Will that be robust against every time that might be seen?